resources/base/kustomization.yaml

```diff
@@ -4,3 +4,4 @@ kind: Kustomization
 
 resources:
   - crds/minio.min.io_tenants.yaml
+  - deployment.yaml
```

Comment:
I think you don't need to move this deployment a level up to apply an overlay just for the images, are you tweaking any other parameter? if not, you can just apply images
```
  
  images:
    - name: minio/operator
      newName: minio/operator
      newTag: dev
```

Reply:
I did this to fix an issue on my mac, but ubuntu virtual machine works differently on github actions, and I haven't been able to replicate that environment locally, so still thinking how to solve current problem. I will try to apply images as shown above....